### DISCUSSION TITLE
Debugging "file not found"

### DISCUSSION THREAD

**Discussion by nelgin** [dosemu2/dosemu2#2242] [General]
Hi all,

I'm looking at installing a new DOS door program and when I run it, I get the obnoxious error

"File not found in module ZWORD13  at address 00c5:0d4c
The instruction at that address is 

00c5:0d4c 9A8609CE18       call 18CE:0986

I set a breakpoint at the previous instruction at 00c5:0d47

00c5:0d47 9A5A06CE18       call 18CE:065A

I use t to trace into the call but after 2-3 minutes of hitting 't' I'm not really getting anywhere. 

I understand I need to look for ah being set to 3d before an INT21 call and the filename will be in the memory location specified as dx being an offset of ds ? Am I talking out my arse here or am I on the right track?

**Comment by stsp** [comment]
You can set break-points to interrupts,
but in this case you should better use
the `-D9+d` log to see what is it trying
to open. Its like I am solving 90% of
problems by looking into logs, and
maybe 2% with debugger, as the latter
is too difficult and should be avoided
most of the time.

**Reply by nelgin** [reply]
Let me try with -D9+d and see what I get. Thank you.

**Reply by nelgin** [reply]
```
MFS: find_file(): no match: /sbbs/xtrn/zwordz/wordz.dic
MFS: open failed: '/sbbs/xtrn/zwordz/wordz.dic'
```

Ok, that was easy. The file supplied in the zip was zwordz.dic so easy fix. 

Thank you, as always.
